SRM Release Notes - version 1.4 (#3784)
harness/developer-hub: release-notes/service-reliability-management.md

### Fixed issues

- PagerDuty incidents were not captured in the free subscription environment. (SRM-10824)
  
  This issue has been resolved. Now, the PagerDuty incidents are being displayed in the free subscription environment.
  
- The change-event API was significantly slower when compared to other listing APIs, and the initial calls to the change-event API were failing. (SRM-15709)
  
  This issue has been resolved. Now, there is a significant improvement in the API response size and response time.
  
- On the Changes page, the **Clear Filter** button was enabled even when no filters were selected. (SRM-15764)
  
  This issue has been resolved. Now, the **Clear Filter** button is enabled only when the filters are selected.
  
- Selecting a large number of values in the **Services** and **Environment** filters on the Changes page led to error 414 indicating that the URL is too long. (SRM-15765)
  
  This issue is resolved by limiting selections to a maximum of 10 values for **Services** and **Environment** filters.
  

> - An alignment issue in the **Change Event** card has been resolved in this release to ensure a consistent display of information. (SRM-15793)

Reviewer: ```suggestion
- An alignment issue with the **Change Event** card has been fixed to ensure consistent information display. (SRM-15793)
```

> - The Service Level Indicator (SLI) iterator was retrying to create SLI tasks that already existed in the system. (SRM15870)

Reviewer: ```suggestion
- The Service Level Indicator (SLI) iterator attempted to create tasks that already existed. (SRM15870)
```
